src/common/PriorityCache.h

```diff
@@ -68,6 +101,42 @@ namespace PriorityCache {
     // Get the name of this cache.
     virtual std::string get_cache_name() const = 0;
   };
+
+  struct Manager {
+    CephContext* cct = nullptr;
+    PerfCounters* logger;
+    std::unordered_map<std::string, PerfCounters*> loggers;
```

Comment:
suggest transparent comparator for these maps so you can index with `std::string_view`. i.e.

`std::unordered_map<std::string, PerfCounters*, std::less<>> loggers;`

Reply:
See: https://stackoverflow.com/questions/51749081/stdstring-view-and-stdstring-in-stdunordered-set